### PR TITLE
Convert proxy from httplib2 to requests

### PR DESCRIPTION
**Pull request by procrastinatio**
This is a quick and dirty solution, until https://github.com/geoadmin/service-mapproxy is fully validated.

[ogcproxy](http://mf-chsdi3.dev.bgdi.ch/mom_proxy_requests/ogcproxy)

[KML 11 MB](http://mf-chsdi3.dev.bgdi.ch/mom_proxy_requests/ogcproxy?url=https://www.procrastinatio.org/kml/simple_000005.kml)

Intégration dans map.geo.admin.ch:
[map.geo.admin.ch](map3.geo.admin.ch?api_url=//mf-chsdi3.dev.bgdi.ch/mom_proxy_requests/)

### CODE REVIEW

**Comment by procrastinatio** [comment]
@loicgasser LOL

File: chsdi/views/ogcproxy.py [hunk 1]
```diff
@@ -39,20 +41,37 @@ def ogcproxy(self):
...
+                             params=self.request.params,
+                             headers=h,
+                             verify=False)
+            else:
+                resp = requests.post(url,
+                             stream=False,
+                             timeout=DEFAULT_REQUEST_TIMEOUT,
+                             params=self.request.params,
+                             headers=h,
+                             verify=False)
```
Comment (loicgasser) [inline chsdi/views/ogcproxy.py]: not such a good idea

**Comment by loicgasser** [comment]
I think you can remove httplib2 from the requirements.
Anyway, this should be removed soon.

**Comment by oterral** [comment]
This PR is not necessary anymore, isn't it? Can we remove hte ogc proxy ?

**Comment by loicgasser** [comment]
No it s not. We ll remove proxy from chsdi after next deploy.

**Comment by procrastinatio** [comment]
I hold the record of unmerged PRs, though that one was not my idea.

**Pull request closed without merging**